include/boundaryvalues.h

```diff
@@ -0,0 +1,137 @@
+#pragma once
+
+#include <fstream>
+#include <iostream>
+
+// #include <deal.II/numerics/vector_tools.h>
+#include <deal.II/base/function.h>
+#include <deal.II/lac/vector.h>
+
+#include "globalvariables.h"
+
+using namespace dealii;
```

Comment:
It's recommended to avoid defining `using namespace` please use `dealii::fn` to be more precise.